vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -56,6 +54,13 @@ protected void validate() {
 
     void passThroughPresentationValue(LocalDate newPresentationValue) {
         super.setPresentationValue(newPresentationValue);
+
+        if (Objects.equals(newPresentationValue, getEmptyValue())
```

Comment:
**note:** It was hard to find a clean way to get an old value since internal pickers seem to only operate with their presentation value, so I don't check for an old value here. As a consequence, the following JS instructions can be sent even when changing from a non-empty value to an empty value. This shouldn't be a big deal though.